File: 프로그래머스/lv1/42576. 완주하지 못한 선수/완주하지 못한 선수.java
import java.util.Arrays;

class Solution {
    public String solution(String[] participant, String[] completion) {
        String answer = null;
        Arrays.sort(participant);
        Arrays.sort(completion);

        for (int i = 0; i < participant.length - 1; i++) {
            if (!participant[i].equals(completion[i])) {
                answer = participant[i];
                break;
            }
        }

        if (answer == null) {
            answer = participant[participant.length - 1];
        }

        return answer;
    }
}

File: 프로그래머스/lv1/42576. 완주하지 못한 선수/완주하지 못한 선수.js
function solution(participant, completion) {
    var answer = '';
    participant.sort();
    completion.sort();

    for (var i = 0; i < completion.length; i++) {
        if (participant[i] !== completion[i]) {
            answer = participant[i];
            break;
        }
    }

    if(answer === ''){
        answer = participant[participant.length - 1];
    }
    return answer;
}